infrastructure/main.tf

```diff
@@ -127,6 +142,38 @@ resource "azurerm_key_vault_secret" "POSTGRES_DATABASE" {
 }
 # endregion
 
+# region staging DB secrets
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES-USER" {
+  name         = "${var.component}-staging-POSTGRES-USER"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.reform-blob-router-staging-db.user_name}"
+}
+
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES-PASS" {
+  name         = "${var.component}-staging-POSTGRES-PASS"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.reform-blob-router-staging-db.postgresql_password}"
+}
+
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES_HOST" {
+  name         = "${var.component}-staging-POSTGRES-HOST"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.reform-blob-router-staging-db.host_name}"
+}
+
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES_PORT" {
+  name         = "${var.component}-staging-POSTGRES-PORT"
+  key_vault_id = "${data.azurerm_key_vault.reform_scan_key_vault.id}"
+  value        = "${module.reform-blob-router-staging-db.postgresql_listen_port}"
+}
+
+resource "azurerm_key_vault_secret" "STAGING_POSTGRES_DATABASE" {
+  name         = "${var.component}--stagingPOSTGRES-DATABASE"
```

Comment:
will those `resource` values and names be used by jenkins pipeline or any other automatic process? would be really nice to sort those names out for once:

- correct usage of case
- correct usage of `_` vs `-`
- no need to reference type of db e.g. `POSTGRES`

i know it is hardcoded for main pipelines we have but if we are free to fix it here - i suggest to do it

Reply:
keeping those consistent with what we have

Reply:
i know about consistency ;). just saying politely suggesting it is not correct. which is not your fault anyway

Reply:
sure will do it :)